docs/summary.md

```diff
@@ -1,7 +1,7 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Example Guide](../examples/example.md)
+* [Crossplane Provider IONOS Cloud - Example (PoC)](../examples/example.md)
```

Comment:
Isn't the usage of `IONOS Cloud` and both `Example` and `PoC` a bit redundant

Reply:
I can update it to "IONOS Cloud POC" if that sounds better?

Reply:
Done, thank you.